dll/win32/shell32/folders/CControlPanelFolder.cpp

```diff
@@ -699,7 +699,7 @@ HRESULT WINAPI CCPLItemMenu::InvokeCommand(LPCMINVOKECOMMANDINFO lpcmi)
 
     TRACE("(%p)->(invcom=%p verb=%p wnd=%p)\n", this, lpcmi, lpcmi->lpVerb, lpcmi->hwnd);
 
-    if (lpcmi->lpVerb == MAKEINTRESOURCEA(IDS_OPEN)) //FIXME
+    if (lpcmi->lpVerb == MAKEINTRESOURCEA(0))
```

Comment:
this "0" (and "1" in the other part of the diff) appear at first sight "arbitrary"/"magic". What do they correspond to?

Reply:
index 0 and 1 of the values added to the menu before?
As documented on MSDN:
https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-icontextmenu-invokecommand

Reply:
thanks for the link